docs/source/conf.py

```diff
@@ -0,0 +1,55 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath('../../'))
+from emissionsapi import *  # noqa
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Emissions API'
+copyright = '2019, info@emissions-api.org'
+author = 'info@emissions-api.org'
```

Comment:
On other places we usually use:

https://github.com/emissions-api/emissions-api/blob/01eaa67c510cd2140f5f35088018f2222c782e28/LICENSE#L3

Reply:
Should be the same now.